plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/Fill.java

```diff
@@ -49,6 +56,29 @@ public void setLatLngs(List<List<LatLng>> latLngs) {
     geometry = Polygon.fromLngLats(points);
   }
 
+  /**
+   * Get a list of lists of LatLng for the fill, which represents the locations of the fill on the map
+   *
+   * @return a list of a lists of the locations of the polygon in a latitude and longitude pairs
+   */
+  @Nullable
+  public List<List<LatLng>> getLatLngs() {
+    Polygon polygon = (Polygon) geometry;
+    List<List<Point>> coordinates = polygon.coordinates();
+    if (coordinates != null) {
+      List<List<LatLng>> latLngs = new ArrayList<>();
+      for (List<Point> innerPoints : coordinates) {
+        List<LatLng> innerList = new ArrayList<>();
+        for (Point point : innerPoints) {
+          innerList.add(new LatLng(point.latitude(), point.longitude()));
+        }
+        latLngs.add(innerList);
+      }
+      return latLngs;
+    }
+    return null;
```

Comment:
do we want nullability for this? Returning an empty list would suffice and would make the method `@NonNull`

Reply:
Good point